tools/fury.js

```diff
@@ -1,80 +1,239 @@
+/*
+  Example script to have Fury parse an API Blueprint.
+*/
+/* eslint no-var:0, func-names:0, no-console:0 no-process-exit:0 */
+var async = require('async');
 var fs = require('fs');
 var path = require('path');
 var parser = require('../lib/fury').legacyBlueprintParser;
-var async = require('async');
+var yargs = require('yargs')
+  .usage('Usage: $0 [options] file|directory')
+  .example('$0 --print example.apib', 'Process a file and print output')
+  .example('$0 -r ./my/examples/', 'Process a directory (recursively)')
+  .example('$0 ./my/examples/ --stats', 'Show processing stats')
+  .options('b', {
+    alias: 'benchmark',
+    describe: 'Show file timing information',
+    type: 'boolean',
+    'default': false
+  })
+  .options('p', {
+    alias: 'parallel',
+    describe: 'Number of files to process in parallel',
+    'default': 1
+  })
+  .options('print', {
```

Comment:
I had already used the `-p` option for the number of parallel jobs. Wasn't sure what to use for `--print` and I figured it's a less likely to be used option since the tool is mostly for testing successful parsing without errors.